src/web_interface/templates/database/database_binary_search.html

```diff
@@ -22,9 +22,19 @@ <h2>Binary Search</h2>
                     for (i = 0; i < tablinks.length; i++) {
                         tablinks[i].className = tablinks[i].className.replace(" active", "");
                     }
-                    document.getElementById(tabname).style.display = "block";
+                    document.getElementById(tabname).style.display = "inline";
```

Comment:
As always I won't code review the javascript mess. The functional test was successful though.